RELEASE_NOTES.rst

```diff
@@ -21,6 +21,279 @@
+- Filtering datasets by recent update events (#26942)
+- Support Is /not Null filter for value is None on webui (#26584)
+- Add search to datasets list (#26893)
+- Split out and handle 'params' in mapped operator (#26100)
+- Add authoring API for TaskGroup mapping (#26844)
+- Add ``one_done`` trigger rule (#26146)
+- Create a more efficient  airflow dag test command that also has better local logging (#26400)
+- Support add/remove permissions to roles commands (#26338)
+- Auto tail file logs in Web UI (#26169)
+- Add triggerer info to task instance in API (#26249)
+- Flag to deserialize value on custom XCom backend (#26343)
+
+Bug Fixes
+^^^^^^^^^
+- Redirect to home view when there are no valid tags in the URL (#25715)
+- Make MappedTaskGroup depend on its expand inputs (#27876)
+- Make DagRun state updates for paused DAGs faster (#27725)
+- Don't explicitly set include_examples to False on task run command (#27813)
+- Fix menu border color (#27789)
+- Fix  backfill  queued  task getting reset to scheduled state.  (#23720)
+- Fix clearing child dag mapped tasks from parent dag (#27501)
+- Handle json encoding of ``V1Pod`` in task callback (#27609)
+- Fix ExternalTaskSensor can't check zipped dag (#27056)
+- Avoid re-fetching DAG run in TriggerDagRunOperator (#27635)
+- Continue on exception when retrieving metadata (#27665)
+- Fix double logging with some task logging handler (#27591)
+- External task sensor fail fix (#27190)
+- Replace FAB url filtering function with Airflows (#27576)
+- Fix mini scheduler expansion of mapped task  (#27506)
+- Add the default None when pop actions (#27537)
+- Display parameter values from serialized dag in trigger dag view. (#27482)
+- Fix getting the dag/task ids from base executor (#27550)
+- Fix sqlalchemy primary key black-out error on DDRQ (#27538)
+- Move TriggerDagRun conf check to execute (#27035)
+- SLAMiss is nullable and not always given back when pulling task instances (#27423)
+- Fix behavior of ``_`` when searching for DAGs (#27448)
+- Add case insensitive constraint to username (#27266)
+- Fix python external template keys (#27256)
+- Resolve trigger assignment race condition (#27072)
+- Update google_analytics.html (#27226)
+- Fix IntegrityError during webserver startup (#27297)
+- reduce extraneous task log requests (#27233)
+- Make RotatingFilehandler used in DagProcessor non-caching (#27223)
+- set executor.job_id to BackfillJob.id for backfills (#27020)
+- Fix some bug in web ui dags list page (auto-refresh & jump search null state) (#27141)
+- Listener: Set task on sqlalchemy taskinstance object (#27167)
+- fix yarn warning from d3-color (#27139)
+- Don't overwrite connection extra with invalid json (#27142)
+- Simplify origin string cleaning (#27143)
+- Don't re-patch pods that are already controlled by current worker (#26778)
+- Add separate error handler for 405(Method not allowed) errors (#26880)
+- Avoid 500 on dag redirect (#27064)
+- Make tracebacks opt-in (#27059)
```

Comment:
```suggestion
```

Hmm, these were already in 2.4.2? Might be others as well.

Reply:
Yeah. The changelog command was getting everything since 2.4.1. I have manually sorted the bugfixes